operators/endpointmetrics/controllers/observabilityendpoint/metrics_collector.go

```diff
@@ -314,6 +322,22 @@ func createDeployment(params CollectorParams) *appsv1.Deployment {
 			})
 	}
 
+	if hubMetricsCollector {
+		//to avoid hub metrics collector from sending status
+		metricsCollectorDep.Spec.Template.Spec.Containers[0].Env = append(metricsCollectorDep.Spec.Template.Spec.Containers[0].Env,
+			corev1.EnvVar{
+				Name:  "STANDALONE",
+				Value: "true",
+			})
+		//Since there is no obsAddOn for hub-metrics-collector, we need to set the resources here
+		metricsCollectorDep.Spec.Template.Spec.Containers[0].Resources = corev1.ResourceRequirements{
+			Limits: corev1.ResourceList{
+				corev1.ResourceMemory: resource.MustParse("100Mi"),
+				corev1.ResourceCPU:    resource.MustParse("10m"),
```

Comment:
I think the resource should be from mco spec instead of hardcode here. Does it make sense?

Reply:
Yes but if there is no obsaddon created for local-cluster, how would it pick the default mco resources?